channeldb/payments.go

```diff
@@ -101,9 +101,9 @@ const (
 	// payment.
 	FailureReasonError FailureReason = 2
 
-	// FailureReasonIncorrectPaymentDetails indicates that either the hash
-	// is unknown or the final cltv delta or amount is incorrect.
-	FailureReasonIncorrectPaymentDetails FailureReason = 3
+	// FailureReasonPaymentDetails indicates that either the hash is unknown
```

Comment:
too long for what? :P

Reply:
for everything 😂but primarily as being an effective identifier, considering our mile wide indentation policy